### ISSUE TITLE
Can not operate "save as" dialog box at Git history window on Rstudio Server

### ISSUE THREAD

**Issue by ujtwr** [rstudio/rstudio#9042]
environment:
RStudio Server Version 1.3.1093
Browser: Firefox, Chrome, Edge (Chrome ver.)

When I tried to revert a file to old version using Git, I accessed to Git history dialog.  Clicked "View file @xxxxx" , the old version file is viewd in the sub window. The sub window has buttons, "Save As", "Print" and "Find".

Clicked the "Save As" button, I can not operate anything.

I  investigated by using Chrome developer tools, the "Save As" dialog box was hided by the sub window at which the old version code.

So I tried to operate by using "Tab key" etc., old version could be saved.

in short, the issue is  sub windows or dialog boxs order of z-index in the browser window.

![screen_shot](https://user-images.githubusercontent.com/42717412/109393665-6c910300-7966-11eb-9532-d8cc2542c3b8.png)
